go/kbfs/libkbfs/config_local.go

```diff
@@ -916,10 +917,11 @@ func (c *ConfigLocal) MakeLogger(module string) logger.Logger {
 
 // MakeVLogger implements the logMaker interface for ConfigLocal.
 func (c *ConfigLocal) MakeVLogger(module string) *libkb.VDebugLog {
-	// No need to lock since c.loggerFn is initialized once at
-	// construction. Also resetCachesWithoutShutdown would deadlock.
```

Comment:
This `resetCachesWithoutShutdown` deadlock is no longer an issue?

Reply:
That comment had been copy/pasted from the function above this one, and doesn't apply to it.